### DISCUSSION TITLE
Ev iş arası yol sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#189] [Zeka Sorusu]
Üyemizin biri evinden işine yürüyerek gidip arabayla dönerse 1,5 saat sürüyor. Hem giderken hem dönerken araba kullanırsa bu süre 30 dakikaya iniyor.
Yürüyerek işe gidip dönmesi ne kadar sürer?

**Comment by Sadik1Dev** [comment]
aynı mesafe yürüyerek t zamanda gidilse araba ile a zamanda geri dönülse ilk senaryoda gidiş geliş süresi toplamı 1,5 saat yani 90 dakikadır.
t+a=90 dakika
ikinci senaryoda aynı mesafe hem gidiş hem dönüş olarak arabayla yapılınca toplam süre: 30 dakika yani 
a+a=30 dakika.
buradan araba ile bir gidişin 15 dakika olduğu ortaya çıkar.
a=15 dakika.
t+a=90'dan t=75 yürüyerek işe gidiş süresi. hem gidiş hem geliş 2t den =150 dakika= 2.5 saat.
